src/models/ratingcourse.model.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../db');
+const logger = require('../helpers/logger');
+
+const RatingCourse = {
+  async create(obj) {
+    const text = 'INSERT INTO ratingcourse(`iduser-ratingcourse`, `idcourse-ratingcourse`, `value-ratingcourse`) VALUES($1, $2, $3) RETURNING *;';
```

Comment:
The quotes you are using dont work. You should use " instead of `.

Reply:
Fixed